### DISCUSSION TITLE
全流程本地部署

### DISCUSSION THREAD

**Discussion by Richard-mei** [idootop/mi-gpt#171] [General]
现在的方案感觉延迟有点高，而且有种跟小爱冲突的感觉，有没有方案/想法不用小爱音箱而且将整个流程在本地部署的？

**Comment by idootop** [comment]
没理解，什么叫做“不用小爱音响”的本地部署

**Reply by idootop** [reply]
简而言之，你需要的是一个完全开放的硬件，唯有刷机。

**Reply by Richard-mei** [reply]
就是本地部署tts stt llm，我看到有类似的项目，但不是小爱这种语音唤醒然后交流的

**Reply by Richard-mei** [reply]
比如这个voicechat2

**Reply by idootop** [reply]
感兴趣你可以自己搞搞，没什么难度。
